lukaszgardecki/library-app
# [FEATURE][F-62] Amin Panel - Add a User List

## 🚀 What?
I have added an Admin Panel page and made updates to the main page to include a button that redirects to this panel. Additionally, I implemented a section that displays all registered users in the system, with each user's name being clickable (leading to a detailed user page that is currently not yet implemented).

## 🎯 Why?
The purpose of these changes is to introduce administrative functionality to the application, allowing administrators to manage users and oversee the platform. This is a foundational step towards building out the full capabilities of the admin section.

## 🛠️ How?
- Created a new view for the Admin Panel.
- Updated the main page by adding a button that redirects to the Admin Panel, which is only visible if the logged-in user has the role 'ADMIN'.
- Implemented a tab within the Admin Panel that lists all created users. Each user name is currently a link to a detailed user page, which will be implemented in a future update.

## ✅ Testing?
1. Log in using the admin test account (username: `admin@example.com`, password: `adminpass`).
2. Click `Administrator` in the top navigation bar – you should be redirected to the Admin Panel.
3. Click `Users` – the list of currently registered users should be displayed.

## 📸 Screenshots
![obraz](https://github.com/user-attachments/assets/d2fd4e6a-ff2a-460c-ba94-1ee67b589dcb)

![obraz](https://github.com/user-attachments/assets/69ffd038-aa93-40fe-bef9-15c9dcbc34de)
